libs.playbook.yml

```diff
@@ -39,6 +39,7 @@ site:
 antora:
   extensions:
     - require: ./boost.js
+    - require: '@antora/lunr-extension'
```

Comment:
I think just 

```yml
    - ./boost.js
    - '@antora/lunr-extension'
```

(no require)

should be fine.

Reply:
Yes as long as we have no configuration keys to pass https://gitlab.com/antora/antora-lunr-extension#user-content-index-only-the-latest-version
using map syntax will keep the consistency.